src/collections/_documentation/guides/migration/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Moving on Up - Switch to Hosted Sentry
+sidebar_order: 6
+---
+
+Sentry offers a hosted cloud solution in addition to an open-source one, both are functionally the same. Despite open-source being a core value for us at Sentry, it isn't necessarily recommended for everyone. As Sentry evolves, many customers are finding out that hosting Sentry on-premise can quickly become expensive to maintain, scale, and support, making our SaaS product, the better and less costly option.
+
+For more information take a look at our post on [Self Hosted or Cloud Sentry?](https://sentry.io/_/resources/about-sentry/self-hosted-vs-cloud/)
```

Comment:
Just read this: https://xscode.com/686/open-source-is-free-but-expensive/

Might be worth referencing it too as this is written by an independent party. We can also refer to this from the "self-hosted vs cloud" page.